language/en/info_acp_ideas.php

```diff
@@ -22,8 +22,10 @@
 	// ACP module
 	'ACP_PHPBB_IDEAS'			=> 'phpBB Ideas',
 	'ACP_PHPBB_IDEAS_SETTINGS'	=> 'Ideas settings',
+	'ACP_PHPBB_IDEAS_SETUP_UTILITIES'	=> 'Ideas setup utilities',
 
 	// ACP Logs
 	'ACP_IDEA_TITLE_EDITED_LOG'			=> '<strong>Idea title edited</strong><br />» phpBB Ideas ID #%s',
 	'ACP_PHPBB_IDEAS_SETTINGS_LOG'		=> '<strong>phpBB Ideas settings changed</strong>',
+	'ACP_PHPBB_IDEAS_FORUM_SETUP_LOG'	=> '<strong>phpBB Ideas forum setup applied</strong>',
```

Comment:
I would remove the _PHPNN_ part from these lang keys as all the other keys are simply ACP_IDEAS_